pgcontents/tests/test_pgmanager.py

```diff
@@ -121,6 +121,24 @@ def test_modified_date(self):
         renamed = cm.get(new_path)
         self.assertGreater(renamed['last_modified'], saved['last_modified'])
 
+    def test_get_file_id(self):
+        cm = self.contents_manager
+
+        # Create a new notebook.
+        nb, name, path = self.new_notebook()
+        model = cm.get(path)
+
+        # Make sure we can get the id and it's not none.
+        id = cm.get_file_id(path)
```

Comment:
Style nit: `id` is [a python built-in](https://docs.python.org/2/library/functions.html#id).  It's generally considered bad to have variables with the same names as the builtins, because if someone tries to use the builtin later in the function, they end up actually using your variable.  A common thing to do here is to add an `_` as a postfix, i.e. `id_` instead of `id`.